testing/resources/docker_hooks_repo/hooks.yaml

```diff
@@ -4,6 +4,12 @@
     language: docker
     files: \.txt$
 
+-   id: docker-hook-arg
+    name: Docker test hook
+    entry: echo -n
```

Comment:
echo -e is probably better (and then include a `r'\n'` and make sure it goes through.  echo -n is kinda difficult to determine in the tests (since it just does `in` checks)